src/Symfony/Installer/NewCommand.php

```diff
@@ -356,6 +358,22 @@ private function cleanUp()
     {
         $this->fs->remove(dirname($this->compressedFilePath));
 
+        try {
+            $commonFiles = array($this->projectDir.'/LICENSE', $this->projectDir.'/UPGRADE.md');
+            $upgradeFiles = glob($this->projectDir.'/UPGRADE-*.*.md');
+            $changelogFiles = glob($this->projectDir.'/CHANGELOG-*.*.md');
```

Comment:
You are right. There's no `CHANGELOG` files because the process that builds the ZIP/TGZ files removes them. But this single line of code will prevent us from future errors in case the ZIP/TGZ builder fails or changes.